angular/tasks/config/vendor.yml

```diff
@@ -2,7 +2,7 @@
 path: node_modules
 js:
   - bowser/bowser.js
-  - moment/moment.js
+  - moment/min/moment-with-locales.min.js
```

Comment:
Hm. This adds about 200k to our payload and I'm not convinced it's worth it.

![screen shot 2017-11-01 at 3 33 11 pm](https://user-images.githubusercontent.com/750477/32257454-028ee7ec-bf1a-11e7-8c6f-8da061649cc4.png)